packages/react/src/components/fileInput/index.ts

```diff
@@ -0,0 +1 @@
+export * from './FileInput';
```

Comment:
Should we only export the component here, and leave the _formatBytes_ without export 🤔